app/services/forest_liana/query_stat_getter.rb

```diff
@@ -11,7 +9,7 @@ def initialize(params)
     def perform
       raw_query = @params['query'].strip
 
-      check_query(raw_query)
+      LiveQueryChecker.check_query(raw_query)
```

Comment:
What about?
```ruby
LiveQueryChecker.check_query(raw_query_to_check) # NOTICE: This query checker checks the query to check. 
```
😝

Seriously, `LiveQueryChecker.validate` or  `LiveQueryChecker.check` is simpler, no?